views/random-beer.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="r-container">
+<div class="r-content">
+
+
+    <img id="random-img" src="{{randomBeer.image_url}}" alt="image missing for {{randomBeer.name}}">
+
+    <h1>{{randomBeer.name}}</h1>
+    <h5>"{{randomBeer.tagline}}"</h5>
+    <p>{{randomBeer.description}}</p>
+
+    <ul>
+    <h6>Food Pairing:</h6>
+        {{#each randomBeer.food_pairing}}
+
+        <li>{{this}}</li>
+
+        {{/each}}
+    </ul>
+
+</div>
+</div>
```

Comment:
You could wrap this entire code with a `{{#with randombeer}}`. That way you would not have to add randombeer everywhere you access it. e.g. instead of "{{ randomBeer.food_pairing}} "{{, just "food_pairing}}".